tests/manage/z_cluster/nodes/test_nodes_maintenance.py

```diff
@@ -182,7 +182,7 @@ def test_2_nodes_maintenance_same_type(self, nodes_type):
 
     @tier2
     @pytest.mark.polarion_id("OCS-1274")
-    def test_2_nodes_different_types(self, pvc_factory, pod_factory):
+    def test_2_nodes_different_types(self, pvc_factory, pod_factory, dc_pod_factory):
```

Comment:
wouldn't it be better to replace pod_factory with dc_pod_factory?

Reply:
it would be good we keep both as they act differently from each other